docs/elasticsearch-spec.asciidoc

```diff
@@ -122,7 +125,9 @@ spec:
         storageClassName: standard
 ----
 
-IMPORTANT: Depending on the Kubernetes configuration and the underlying file system, some persistent volumes cannot be resized after they are created. You should consider future storage requirements when defining volume claims to ensure that you have plenty of space available to sustain the expected growth.
+ECK automatically deletes VolumeClaimTemplates resources if they are not required for any Elasticsearch node. The corresponding PersistentVolume may be preserved, depending on the configured link:https://kubernetes.io/docs/concepts/storage/storage-classes/#reclaim-policy[storage class reclaim policy].
```

Comment:
If VolumeClaimTemplates refers to the code sample above, I would write it like this:
ECK automatically deletes `volumeClaimTemplates` resources

Code type font for `PersistentVolume` too.

Reply:
We use the templates to create actual volume claims, which is what we delete. Also, we have not been using code fonts for resource names in the past (or even for say, StatefulSets in this document), should we start now?